src/main/java/bibliotecame/back/User/UserController.java

```diff
@@ -36,21 +36,20 @@ public UserController(UserService userService){
 
     @DeleteMapping("deleteUser/{id}")
     public ResponseEntity<Integer> deleteUser(@PathVariable Integer id){
-//
-//        UserModel user;
-//        try {
-//            user = this.userService.findLogged();
-//        } catch (RuntimeException e){
-//            return new ResponseEntity<>(id, HttpStatus.BAD_REQUEST);
-//        }
-//
-//        if(user.getId() != id){
-//            return new ResponseEntity<>(id, HttpStatus.UNAUTHORIZED);
-//        }
+
+        UserModel user;
+        try {
+            user = this.userService.findLogged();
+        } catch (RuntimeException e){
+            return new ResponseEntity<>(id, HttpStatus.BAD_REQUEST);
+        }
+
+        if(user.getId() != id){
+            return new ResponseEntity<>(id, HttpStatus.UNAUTHORIZED);
+        }
```

Comment:
This seems to not been working.

Test done: 
- Log in with some user (on the frontend)
- Try to delete the user (on the frontend). (the id on the URL is the same as the user id)
- The request failed (on the backend seems to no be finding the logged user)

Reply:
The real problem was the content-type. Resolved.